google-cloud-vision/samples/product_search/acceptance/product_search_import_product_images_test.rb

```diff
@@ -27,16 +27,31 @@
       "fake_product_id_for_testing_2" => "shoes_2.jpg"
     }
 
-    output = `ruby #{snippet_filepath} #{@project_id} #{@location}`
+    product_set = nil
+    error = nil
+    5.times do
+      next if product_set
```

Comment:
just use `break` to break out of the loop, instead of `next` which keeps iterating.